Utils/gtag.js

```diff
@@ -0,0 +1,17 @@
+export const GA_TRACKING_ID = "GTM-WVFLH9>";
```

Comment:
Should we put this value into a more secure location? I guess something like a `.env` file.

Reply:
I would not think that will be necessary as it is usually exposed in the head of the page. Moreso, it is completely useless to anyone that gets hold of it.

<img width="470" alt="GTA" src="https://user-images.githubusercontent.com/20474902/94908884-3d02cb00-049a-11eb-94d8-657af92f0d3c.png">

Reply:
Ah yes, good point. 👍 
Actually I see that it appears to be using the MAS value, though that is `GTM-WVFLH9` not `GTM-WVFLH9>`. Can we just check we have the correct one here?

Reply:
Ah! Dobrin gave that to me but I think your must be the correct one since that is already in production. Let me confirm and make the necessary changes if needed.

Reply:
Fixed. That was an error on my side.

Reply:
👍